packages/cli/src/sass/mixins.ts

```diff
@@ -76,7 +76,7 @@ $displayVar: null !default;
         ($subset == $unicodeSubset) or
           (
             // Is numeric subset
```

Comment:
Could you update this comment to explain what's going on?

Reply:
Done, and it also made me realize that I missed some use-case. I added some more test to cover that as well